E1-basic-exercises/src/main/java/com/redelles/e1/E1MainProgram.java

```diff
@@ -9,6 +10,12 @@
 
     public static void main(String[] args) throws IOException {
         System.out.println("E1. Odd numbers between 10 to 101");
+
+        for (Integer index = 10; index <= 101; index++){
+            if (index % 2 != 0){
```

Comment:
You should use the method Odd.odd() to know if the number is odd. Change in the rest of the methods